packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -139,7 +147,13 @@ class CodePushClient {
     final response = await _httpClient.send(request);
     final body = await response.stream.bytesToString();
 
-    if (response.statusCode != HttpStatus.ok) throw _parseErrorResponse(body);
+    if (response.statusCode == HttpStatus.conflict) {
+      throw ResourceConflictException(
```

Comment:
Thoughts on adjusting to:

```dart
class CodePushConflict extends CodePushException {
  CodePushConflict({super.message, super.details});
}
```

And adjusting `_parseErrorResponse` to handle conflicts in a generic way?

Reply:
See https://github.com/shorebirdtech/shorebird/pull/561